pkg/apiserver/api_installer.go

```diff
@@ -148,7 +148,7 @@ func (a *APIInstaller) registerResourceHandlers(path string, storage rest.Storag
 		if err != nil {
 			return nil, err
 		}
-		parentMapping, err := a.group.Mapper.RESTMapping(parentKind, a.group.Version)
+		parentMapping, err := a.group.Mapper.RESTMapping(parentKind, a.group.GroupVersion.String())
```

Comment:
> Do you want to change the signature of RESTMapping to use GroupVersion instead of string?

Also,  yes, that's on my list.  I'm trying to keep these pieces small though..